src/contracts/ValidatorManager/SecurityModule/PoASecurityModule.sol

```diff
@@ -17,27 +17,33 @@ import {Ownable} from "@openzeppelin/contracts@5.0.2/access/Ownable.sol";
 /**
  * @dev Implementation of the {IPoAValidatorManager} interface.
  *
- * @custom:security-contact https://github.com/ava-labs/teleporter/blob/main/SECURITY.md
+ * @custom:security-contact security@suzaku.network
  */
 contract PoASecurityModule is IPoAValidatorManager, Ownable {
+    error ZeroAddress();
+
     IBalancerValidatorManager public immutable balancerValidatorManager;
 
     constructor(address balancerValidatorManager_, address initialOwner) Ownable(initialOwner) {
```

Comment:
Should this check also exist on InitialOwner not to be zero address?

Reply:
Already done in `Ownable`